server/graphql/v2/input/UpdateAttributesInput.js

```diff
@@ -0,0 +1,19 @@
+import { GraphQLBoolean, GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import { IsoDateString } from '../../v1/types';
+
+import { AccountReferenceInput } from './AccountReferenceInput';
+
+export const UpdateAttributesInput = new GraphQLInputObjectType({
+  name: 'UpdateAttributesInput',
```

Comment:
Naming convention for this one would be:
```suggestion
export const UpdateUpdateInput = new GraphQLInputObjectType({
  name: 'UpdateUpdateInput',
```

I agree it looks really strange :smile: